ntoskrnl/ex/sysinfo.c

```diff
@@ -562,25 +562,26 @@ NtEnumerateSystemEnvironmentValuesEx(IN ULONG InformationClass,
     return STATUS_NOT_IMPLEMENTED;
 }
 
+_Success_(NT_SUCCESS(return))
```

Comment:
Isn't it already understood? (annotation included in the typedef of NTSTATUS)

Reply:
Yes it is, thanks for your infomation, superfluous SAL notations are removed.